policies/vendors/chainguard/policies_test.go

```diff
@@ -16,50 +16,59 @@ func TestPolicies(t *testing.T) {
 		name   string
 		policy string
 		image  string
-
-		check Check
-	}{{
-		name:   "static-is-signed",
-		policy: "chainguard-images-signed.yaml",
-		image:  "cgr.dev/chainguard/static",
-		check:  All(NoErrors, NoWarnings),
-	}, {
-		name:   "busybox-is-signed",
-		policy: "chainguard-images-signed.yaml",
-		image:  "cgr.dev/chainguard/busybox",
-		check:  All(NoErrors, NoWarnings),
-	}, {
-		name:   "static-is-attested",
-		policy: "chainguard-images-attested-cue.yaml",
-		image:  "cgr.dev/chainguard/static",
-		check:  All(NoErrors, NoWarnings),
-	}, {
-		name:   "busybox-is-attested",
-		policy: "chainguard-images-attested-cue.yaml",
-		image:  "cgr.dev/chainguard/busybox",
-		check:  All(NoErrors, NoWarnings),
-	}, {
-		name:   "old-image-does-not-verify",
-		policy: "chainguard-images-signed.yaml",
-		// predated the mono repo, so the signer is wrong.
-		image: "cgr.dev/chainguard/static@sha256:a9650a15060275287ebf4530b34020b8d998bd2de9aea00d113c332d8c41eb0b",
-		check: All(CheckError("none of the expected identities matched"), NoWarnings),
-	}, {
-		name:   "chainctl is signed",
-		policy: "chainguard-enforce-agent-signed.yaml",
-		image:  "us.gcr.io/prod-enforce-fabc/chainctl@sha256:1792f61020540158708182d3cebd17f20c65b5866c803e62d5539b8e53c860f5",
-		check:  All(NoErrors, NoWarnings),
-	}, {
-		name:   "agent is signed",
-		policy: "chainguard-enforce-agent-signed.yaml",
-		image:  "us.gcr.io/prod-enforce-fabc/controlplane@sha256:25ab95c63c7148dd7ba5d8de82abcc67a9fea9e9d2be74bb675a75f4f31e3762",
-		check:  All(NoErrors, NoWarnings),
-	}, {
-		name:   "tagged chainctl is signed",
-		policy: "chainguard-enforce-agent-signed.yaml",
-		image:  "us.gcr.io/prod-enforce-fabc/chainctl:v0.1.117",
-		check:  All(NoErrors, NoWarnings),
-	}}
+		check  Check
+	}{
+		{
+			name:   "static-is-signed",
+			policy: "chainguard-images-signed.yaml",
+			image:  "cgr.dev/chainguard/static",
+			check:  All(NoErrors, NoWarnings),
+		},
+		{
+			name:   "busybox-is-signed",
+			policy: "chainguard-images-signed.yaml",
+			image:  "cgr.dev/chainguard/busybox",
+			check:  All(NoErrors, NoWarnings),
+		},
+		{
+			name:   "static-is-attested",
+			policy: "chainguard-images-attested-cue.yaml",
+			image:  "cgr.dev/chainguard/static",
+			check:  All(NoErrors, NoWarnings),
+		},
+		// TODO: uncomment this when we start signing the vuln attestation again
```

Comment:
I was more into keeping this test but commenting the attestation check for vulnerabilities, 

```
apiVersion: policy.sigstore.dev/v1beta1
kind: ClusterImagePolicy
metadata:
  name: chainguard-images-attested-cue
  annotations:
    catalog.chainguard.dev/title: Chainguard Images Attested
    catalog.chainguard.dev/description: Enforce Chainguard images have SPDX SBOM, vuln and SLSA provenance attestations.
    catalog.chainguard.dev/labels: chainguard
spec:
  images:
    - glob: cgr.dev/chainguard/**
  authorities:
    - keyless:
        url: https://fulcio.sigstore.dev
        identities:
          - issuer: https://token.actions.githubusercontent.com
            subject: https://github.com/chainguard-images/images/.github/workflows/release.yaml@refs/heads/main
      ctlog:
        url: https://rekor.sigstore.dev
      attestations:
        - name: must-have-spdx-attestation-cue
          predicateType: https://spdx.dev/Document
          policy:
            type: cue
            data: |
              predicateType: "https://spdx.dev/Document"
        - name: must-have-provenance-attestation-cue
          predicateType: https://slsa.dev/provenance/v0.2
          policy:
            type: cue
            data: |
              predicateType: "https://slsa.dev/provenance/v0.2"
              ```

Reply:
thanks for the review, done that change PTAL